src/components/_internal/BaseTable/Head/Head.tsx

```diff
@@ -83,4 +83,5 @@ function Head<D extends Record<string, unknown>>({
   );
 }
 
+Head.displayName = Head;
```

Comment:
there are  missing quote marks in this line

Reply:
🤦 fixed